es-rest/storage-es.yml

```diff
@@ -0,0 +1,83 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#cluster:
+#  zookeeper:
+#    hostPort: localhost:2181
+#    sessionTimeout: 100000
+naming:
+  jetty:
+    host: localhost
+    port: 10800
+    contextPath: /
+cache:
+#  guava:
+  caffeine:
+remote:
+  gRPC:
+    host: localhost
+    port: 11800
+agent_gRPC:
+  gRPC:
+    host: localhost
+    port: 11800
+    #Set these two setting to open ssl
+    #sslCertChainFile: $path
+    #sslPrivateKeyFile: $path
+
+    #Set your own token to active auth
+    #authentication: xxxxxx
+agent_jetty:
+  jetty:
+    host: localhost
+    port: 12800
+    contextPath: /
+analysis_register:
+  default:
+analysis_jvm:
+  default:
+analysis_segment_parser:
+  default:
+    bufferFilePath: ../buffer/
+    bufferOffsetMaxFileSize: 10M
+    bufferSegmentMaxFileSize: 500M
+ui:
+  jetty:
+    host: localhost
+    port: 12800
+    contextPath: /
+storage:
+  elasticsearch:
```

Comment:
This is elasticsearch rest storage module implementation, clean this file, make it empty.

Reply:
ok.